src/panel/poi/blocks/Block.jsx

```diff
@@ -22,7 +22,7 @@ const Block = ({
       title={tooltip}
       {...rest}
     >
-      {typeof icon === 'string' ? <i className={`block-icon icon-${icon}`} /> : icon}
+      <div className="block-icon">{icon}</div>
```

Comment:
Nitpick: This class "block-icon" adds an extra margin, which is not suitable for icons used in the "Details" section. See this spectacular Before/After gif:
![Peek 03-08-2021 17-14](https://user-images.githubusercontent.com/4726554/128040762-2c73d9b8-a36c-40a2-866c-499ea8ba319b.gif)